ConnectorRevit/ConnectorRevit/Entry/SpeckleRevitCommand2.cs

```diff
@@ -67,6 +71,55 @@ public Result Execute(ExternalCommandData commandData, ref string message, Eleme
       return Result.Succeeded;
     }
 
+    internal static void RegisterPane()
+    {
+      if (!UseDockablePanel)
+        return;
+
+      var registered = DockablePane.PaneIsRegistered(PanelId);
+      var created = DockablePane.PaneExists(PanelId);
+
+      if (registered && created)
+      {
+        _panel.Init();
+        return;
+      }
+
+      if (!registered)
+      {
+        //Register dockable panel
+        var viewModel = new MainViewModel(Bindings);
+        _panel = new Panel
+        {
+          DataContext = viewModel
+        };
+        App.AppInstance.RegisterDockablePane(PanelId, "Speckle", _panel);
+        _panel.Init();
+      }
+      created = DockablePane.PaneExists(PanelId);
+
+      //if revit was launched double-clicking on a Revit file, we're screwed
+      //could maybe show the old window?
+      if (!created && App.AppInstance.Application.Documents.Size > 0)
+      {
+        TaskDialog mainDialog = new TaskDialog("Dockable Panel Issue");
+        mainDialog.MainInstruction = "Dockable Panel Issue";
+        mainDialog.MainContent =
+            "Revit cannot properly register Dockable Panels when launched double-clicking a Revit file. "
```

Comment:
This is a little nit-picky, but it seems like there should either be 'by' or 'via' between 'launched' and 'double-clicking'

Reply:
I agree 😅